src/Avalonia.Base/Media/TextFormatting/Unicode/BiDi.trie.cs

```diff
@@ -18,7 +18,7 @@ public static UnicodeTrie Trie
         get => new(Data, 0x00100000, 0x00000000);
     } 
     
-    private static ReadOnlySpan<uint> Data => new uint[]
```

Comment:
As discussed in the comments, please revert the changes in all `*.trie.cs` files. Or Use collection expressions (`[ ]`) since they are now available, they will be more obvious than `=> new`.